rwengine/src/script/ScriptTypes.cpp

```diff
@@ -220,6 +220,18 @@ ScriptObjectType<GarageInfo> ScriptArguments::getScriptObject(
     return {param.handleValue(), garage};
 }
 template <>
+ScriptObjectType<Payphone> ScriptArguments::getScriptObject(
+    unsigned int arg) const {
+    auto& param = (*this)[arg];
+    RW_CHECK(param.isLvalue(), "Non lvalue passed as object");
+    auto& payphones = getWorld()->payphones;
+    Payphone* payphone = nullptr;
+    if (size_t(*param.handleValue()) < payphones.size()) {
```

Comment:
size_t is unsigned, also this code was copied from other function, so what? do I change names in all similar functions then?

Reply:
At least here because it's used more than once and it makes it clear it's being used as an index.

And what I mean is the index (`*param.handleValue()`) could be < 0 from the script, before the cast. Should probably be an assertion.

Reply:
How about using ```std::vector::at```